account_financial_risk/models/account_invoice.py

```diff
@@ -82,4 +83,43 @@ def action_post(self):
                     )
                     .action_show()
                 )
-        return super().action_post()
+        return super().post()
```

Comment:
¿Why this change?
¿`def action_post(self):` and `return super().post()`?

Please, include this changes:https://github.com/OCA/credit-control/commit/4bd8fcd1f1cdc8120b354f0054782ecb24659948

Reply:
This wizard call to method post:
https://github.com/odoo/odoo/blob/e3c0edd57c1c2cb6da9e5ce41af9df3fb629193a/addons/account/wizard/account_validate_account_move.py#L20
therefore method post should be extended and not action_post to the warning is displayed when several invoices are validated

Reply:
Ping @psugne https://github.com/OCA/credit-control/commit/5d95c65e3c569426696b7d0400d51566e4105e14#diff-75cb51ce626ac1a0a6744302a700a18a6df917b7f76a8bb9a27a4d363c28e8bbR62

Reply:
thank you @carlosdauden that was a mistake doing the cherrypick. I included 4bd8fcd now. But I am setting this to WIP, I have to test it again.